notebooks/timelapse-camera-tutorial.ipynb

```diff
@@ -41,145 +41,40 @@
     "* Cameras were installed on the east and west side of the Grand Mesa, across a vegetation scale of 1-9, using the convention __XMR__:\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    file_content = files[0].read()
</pre><p>This is prohibitively slow when I try to run it locally. I wonder if we work with a subset of images?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SnowEx/snow-cookbook/pull/28/discussion/#comment-2471207728'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Interesting, because it was just working yesterday. I wonder if it has to do with the system maintenance today. The maintenance lasts until the end of the work day, so I'll check it out tomorrow morning.</p>